cost/aws/reserved_instances/expiration/README.md

```diff
@@ -18,6 +18,13 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer